### DISCUSSION TITLE
FastLogin API

### DISCUSSION THREAD

**Discussion by Hyronymos1337** [games647/FastLogin#1185] [General]
Hi guys,

We are using our own authentication plugin which is currently incompatible with fastlogin.

Is there any API from fastlogin that we can use?

**Comment by TuxCoding** [comment]
You could use the `setAuthPlugin` method to provide your own auth hook implementation.
https://github.com/games647/FastLogin/blob/a65a5f302023ddf93fabf051adf1f7136d417855/core/src/main/java/com/github/games647/fastlogin/core/shared/FastLoginCore.java#L291
